app/src/main/java/com/canopas/yourspace/ui/MainActivity.kt

```
@@ -287,3 +306,28 @@ fun SpaceNotFoundPopup() {
+    val context = LocalContext.current
+
+    AppAlertDialog(
+        title = "Turn off Battery Saver Mode",
+        subTitle = "When Battery Saver is on, GroupTrack cannot access location data, preventing the app from working correctly. Please turn off the power saving mode.",
+        confirmBtnText = "Turn Off",
+        dismissBtnText = stringResource(R.string.common_btn_cancel),
+        onConfirmClick = {
+            try {
+                val intent = Intent(Settings.ACTION_BATTERY_SAVER_SETTINGS)
+                context.startActivity(intent)
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)